package.json

```diff
@@ -10,10 +10,10 @@
   "engines": {
     "node": "8.6.0"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
`chosen`, `jquery`, and presumably `underscore` and `uswds` aren't just `devDependencies` - should we move just `node-sass` to `devDependencies` instead?

Reply:
After reading the documentation on `npm` [here](https://docs.npmjs.com/files/package.json#devdependencies), I think I'll make these `dependencies` again. But follow up on using `.cfignore` to not upload these files to the server. :cloud:.gov should not be leveraging any `npm`-related code to run Tock. All the assets should be copied over to `static` at integration-time.